packages/driver/src/cy/mouse.ts

```diff
@@ -58,6 +58,7 @@ type DefaultMouseOptions = ModifiersEventOptions & CoordsEventOptions & {
 export const create = (state: StateFunc, keyboard: Keyboard, focused: IFocused, Cypress: ICypress) => {
   const isFirefox = Cypress.browser.family === 'firefox'
   const isWebKit = Cypress.isBrowser('webkit')
+  const isChromium116OrLater = Cypress.isBrowser({ family: 'chromium' }) && Cypress.browserMajorVersion() >= 116
```

Comment:
A comment explaining what changed in 116 would be nice

Reply:
Added [974b8f9](https://github.com/cypress-io/cypress/pull/26966/commits/974b8f91cb947337a6e5089954e17bb7a329c07f)